### PR TITLE
feat: add documentation website using Docusaurus.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#564

The documentation website is:
* already deployed at [https://logion-network.github.io/logion-api/](https://logion-network.github.io/logion-api/)
* locally viewable with `cd website ; yarn start`

### CODE REVIEW

File: website/docs/intro.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+---
+
+# Intro
+
+## Install
+
+Install package [`@logion/extension`](https://www.npmjs.com/package/@logion/extension) with your favorite package manager (recommended).
+Note that the extension embeds the [`@logion/client`](https://www.npmjs.com/package/@logion/client).
```
Comment (gdethier) [inline website/docs/intro.md:10]: It does not embed it strictly speaking, it brings it in as a dependency. But I guess it's OK.

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
```
Comment (gdethier) [inline website/docusaurus.config.js:2]: Should we keep those "help" comments (this one and the ones below)?

File: website/src/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    </header>
+  );
+}
+
+export default function Home(): JSX.Element {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+    <Layout
+      title={`${siteConfig.title}`}
+      description="Description will go into a meta tag in <head />">
```
Comment (gdethier) [inline website/src/pages/index.tsx:34]: Set an actual description.

**Review by gdethier — APPROVED** [review]
Good job! For the logo, I would use the same as on our Twitter account: https://twitter.com/logion_network. Also, the links color does not go well with the footer background. But I guess that we should first focus on content and "fix" graphics later.

**Comment by benoitdevos** [comment]
> Good job! For the logo, I would use the same as on our Twitter account: https://twitter.com/logion_network. Also, the links color does not go well with the footer background. But I guess that we should first focus on content and "fix" graphics later.



> 

Links color and logo adapted.

File: website/docs/intro.md (hunk 1 above)
Comment (benoitdevos) [reply website/docs/intro.md]: Wording adapted.

File: website/docusaurus.config.js (hunk 2 above)
Comment (benoitdevos) [reply website/docusaurus.config.js]: Comments removed.

File: website/src/pages/index.tsx (hunk 3 above)
Comment (benoitdevos) [reply website/src/pages/index.tsx]: Done.

**Pull request merged**